### PR TITLE
OPS: Upgrade RN

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4h40689vku5zuk49tnezdrfmf0

[download apk](http://filestorage.bluewallet.io:1488/1584917996.apk)

**Comment by Overtorment** [comment]
Android on appetite doesn’t even start

**Comment by Overtorment** [comment]
App.test.js also fails, smth with the mock

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/haq9tu8kw8hak40wqb3yjx14hr

[download apk](http://filestorage.bluewallet.io:1488/1584919845.apk)

**Comment by Overtorment** [comment]
Noice! Even all tests passed

File: package.json [hunk 1]
```diff
@@ -121,7 +121,7 @@
     "react-native-snap-carousel": "3.8.4",
     "react-native-sortable-list": "0.0.23",
     "react-native-svg": "9.13.6",
-    "react-native-swiper": "git+https://github.com/BlueWallet/react-native-swiper.git#1.5.14",
+    "react-native-swiper": "1.6.0-nightly.5",
```
Comment (Overtorment) [inline package.json]: Oh, they merged your fixes, right?
Dont forget to delete this repo from `BlueWallet/`

**Comment by Overtorment** [comment]
Found only one issue:
On android real devices, wallets carousel is not swipeable. You obviously need to have at least one wallet. Reproduceable on local Android emulator, Browserstack (had to manually upload `1584919845.apk`) but NOT on Appetize. Tested on both my android devices, reproduced ok.

Everything else eems to be worign fine

@marcosrdz I can merge this and you can wor on a fix on `master` or you can fix it in this branch. What say you?

Peek 2020-03-23 11-45

**Pull request merged**